Documentation/nfs.md

```diff
@@ -279,6 +346,64 @@ After the NFS server pod is running, follow the same instructions from the previ
 
 After that, follow the rest of the instructions in the [Accessing the Export](nfs.md#accessing-the-export) section and then the [Consuming the Export](nfs.md#consuming-the-export) section to consume the NFS volume.
 
+## Dynamic provisioning with NFS
+With PR https://github.com/rook/rook/pull/2758 rook starts supporting dynamic provisioning with NFS. This example will be showing how dynamic provisioning feature can be used for nfs.
+
+Once the NFS Operator and an instance of NFSServer is deployed. A storageclass with below example can be created.
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  labels:
+    app: rook-nfs
+  name: rook-nfs-share1
+parameters:
+  claimPath: /<Claim name>
```

Comment:
for the export that this StorageClass is referring to, why do we select by the underlying PVC of the export?  My expectation would be that this would just be the name of the export.  The name of the export is external/user facing, and the underlying PVC of the export feels more like an implementation detail that shouldn't be exposed to the creator of the storageClass.  What was the reasoning here?

This is what I would have expected:
```yaml
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  labels:
    app: rook-nfs
  name: rook-nfs-share1
parameters:
  exportName: <export name>
provisioner: rook.io/<NFSServer Name>
reclaimPolicy: Delete
volumeBindingMode: Immediate
```

Reply:
umm yeah, I agree with you regarding this. Giving the export name makes much more sense, I actually just followed the earlier approach of adding the claimed path. Will be changing it

Reply:
cool addressed this comment and also remove one to one mapping of provisioner and NFS which needs to add 2 more parameters `NFSServerName` and `NFSServerNamespace` along with this.